common/src/service/Resources.Designer.cs

```diff
@@ -0,0 +1,506 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+using System.Reflection;
+
+namespace Microsoft.Azure.Devices.Common {
```

Comment:
We've discussed about this offline: we should remove localization.
Tracked by #177.

/cc @jasmineymlo @tameraw